src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,26 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveArtworkItem: {
+      type: ConversationItemType,
+      resolve: (conversationItem, _args, { artworkLoader }) => {
+        if (conversationItem.item_type === "Artwork") {
+          return artworkLoader(conversationItem.properties.id).then(
+            (artwork) => {
+              const updatedArtwork = {
+                ...artwork,
+                __typename: "Artwork",
+              }
+              return artwork.published ? updatedArtwork : null
+            }
+          )
+        } else if (conversationItem.item_type === "PartnerShow") {
+          throw new Error("PartnerShow not supported.")
```

Comment:
Not sure if this makes sense here but since the field is called `liveArtworkItem` could this just return null? I suppose it depends if the client is sure that it is querying an artwork by this point. We probably don't want to throw an error if we are rendering something that may be an artwork (in which case we want the non-snapshotted version), or a partner show (in which case we don't care).

Reply:
@erikdstock this is a good point. So here is where we are [querying](https://github.com/artsy/force/blob/master/src/v2/Apps/Conversation/Components/Conversation.tsx#L243-L253) for `isOfferableFromInquiry` in force and here is where we could query for it in [eigen](https://github.com/artsy/eigen/blob/master/src/lib/Scenes/Inbox/Components/Conversations/ConversationCTA.tsx#L78-L80). They are both on an Artwork type. But do we feel we need to make the implementation less strict?

Reply:
If anyone were to query for this on PartnerShow it would cause the entire query to fail so that's worth keeping in mind.

Reply:
I'd say if the name says artwork, return null if there is no artwork item. but now clients who know they are working with an artwork must understand that the item may be null (maybe it is anyways, like if the artwork has been unpublished)

Reply:
I think it's safe to return null here because of the naming as you pointed out.